service/http.directory/src/test/java/org/kaazing/gateway/service/http/directory/HttpDirectoryServiceIT.java

```diff
@@ -244,14 +253,9 @@ public void testHexEncodedUri() throws Exception {
         robot.finish();
     }
 
-    @Specification("uri.too.long")
-    @Test(timeout = 5000)
-    public void testUriTooLong() throws Exception {
-        robot.finish();
-    }
-
     @Specification("invalid.uri.with.space")
     @Test(timeout = 5000)
```

Comment:
Please move all the timeouts to use a DisableOnDebug timeout rule.  See https://github.com/kaazing/gateway/blob/4357da20f97e37ae37b524b185b9e9bef74a155d/transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/connector/WsebConnectorIT.java for example